### ISSUE TITLE
Improve experience when downloading songs

### ISSUE THREAD

**Issue by kaneryu** [Malopieds/InnerTune#316]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

There should be an in-app progress bar that shows the process of song downloading, but it only shows when downloading >100 songs

### Why do you want this feature?

Currently, mass downloading songs is guesswork, between some songs not being downloaded, and the process (seemingly) freezing

**Comment by Malopieds** [comment]
You can activate notifications and you will have the remaining number of songs that are going to be downloaded (and the one that failed)

**Comment by kaneryu** [comment]
> You can activate notifications and you will have the remaining number of songs that are going to be downloaded (and the one that failed)

That is true and very useful, but it doesn't show the status of the entire download, and doesn't address how downloads freeze for a while before continuing
